src/Common/src/Interop/Linux/System.Native/Interop.INotify.cs

```diff
@@ -2,28 +2,29 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
 using System.Runtime.InteropServices;
 
 internal static partial class Interop
 {
-    internal static partial class libc
+    internal static partial class Sys
     {
-        [DllImport(Libraries.Libc, SetLastError = true)]
-        internal static extern int inotify_init();
+        [DllImport(Libraries.SystemNative, SetLastError = true)]
+        internal static extern int INotifyInit();
 
-        [DllImport(Libraries.Libc, SetLastError = true)]
-        internal static extern int inotify_add_watch(int fd, string pathname, uint mask);
+        [DllImport(Libraries.SystemNative, SetLastError = true)]
+        internal static extern int INotifyAddWatch(int fd, string pathName, uint mask);
 
-        [DllImport(Libraries.Libc, SetLastError = true, EntryPoint = "inotify_rm_watch")]
-        private static extern int inotify_rm_watch_extern(int fd, int wd);
+        [DllImport(Libraries.SystemNative, SetLastError = true, EntryPoint = "INotifyRemoveWatch")]
+        private static extern int INotifyRemoveWatch_private(int fd, int wd);
```

Comment:
I've gotten the exact opposite feedback in the past:  https://github.com/dotnet/corefx/pull/4088#discussion_r42870449